### ISSUE TITLE
process.extract*: Inconsistent behaviour between python2 and python3 when using pandas.Series

### ISSUE THREAD

**Issue by normanius** [seatgeek/fuzzywuzzy#181]
When executing the following code, `process.extract*` functions do not behave consistently.

```
from fuzzywuzzy import process
import pandas as pd

a = pd.Series.from_array(['abc', 'def', 'ghi'])
process.extractOne('abd', a)
```

In python2 (python 2.7.12, pandas 0.20.3, fuzzywuzzy 0.15.1) the output is:
```
('abc', 67,)
```
In python3 (python 3.5.2, pandas 0.20.3, fuzzywuzzy 0.15.1) the output is:
```
('abc', 67, 0)
```

So apparently (following the documentation in process.p), fuzzywuzzy under python3 interprets `pd.Series` as a dictionary and not as a list-like object. 

Not entirely sure if the problem is on the fuzzy side - but some users might get confused when migrating from python2 to python3.

**Comment by josegonzalez** [comment]
We're unlikely to fix this as all of our applications are python2. That said, I'd be happy to review a pull request that fixes the issue :)

**Comment by normanius** [comment]
The inconsistency indeed originates from pandas (https://github.com/pandas-dev/pandas/issues/13918) and will be solved for future versions of pandas (>=0.21).

No further action required, I guess. Sorry for bothering you. :)

**Comment by josegonzalez** [comment]
No worries, thanks for digging!
